tools/operator/testclient/operations/instantiatechaincode.go

```diff
@@ -281,80 +327,149 @@ func (i InstantiateCCUIObject) queryInstalledusingCLI(instantiateObject Instanti
 func (i InstantiateCCUIObject) approveCCusingCLI(instantiateObject InstantiateCCUIObject) error {
 
 	var packageID string
-	orgName := instantiateObject.ChannelOpt.OrgName[0]
-	currentDir, err := paths.GetCurrentDir()
-	if err != nil {
-		return err
-	}
-	err = SetEnvForCLI(orgName, instantiateObject.ConnProfilePath, instantiateObject.TLS, currentDir)
-	if err != nil {
-		return err
-	}
-	queryInstalled, _ := i.queryInstalledusingCLI(instantiateObject)
-	for j := 0; j < len(queryInstalled.CC); j++ {
-		if queryInstalled.CC[j].Label == fmt.Sprintf("%s_%s", instantiateObject.ChainCodeID, instantiateObject.ChainCodeVer) {
-			packageID = queryInstalled.CC[j].PackageID
-			break
+	var peerAddress string
+	var ordererAddress string
+	var connProfilePath string
```

Comment:
These three can be made local variables: `peerAddress`, `ordererAddress`, `connProfilePath`